### ISSUE TITLE
About ghalint.yaml exclude

### ISSUE THREAD

**Issue by y-miyazaki** [suzuki-shunsuke/ghalint#1342]
I am trying to use GitHub Agentic Workflows, but the automatically generated workflow files are triggering errors when running ghalint run.
https://github.com/y-miyazaki/config/blob/main/.github/workflows/daily-doc-updater.lock.yml

```Bash
$ ghalint run
May 22 12:03:22.367 ERR the job violates policies program=ghalint version=1.5.6 workflow_file_path=.github/workflows/daily-doc-updater.lock.yml policy_name=job_timeout_minutes_is_required reference=https://github.com/suzuki-shunsuke/ghalint/blob/main/docs/policies/012.md job_name=update_cache_memory error="job's timeout-minutes is required"
May 22 12:03:22.367 ERR the job violates policies program=ghalint version=1.5.6 workflow_file_path=.github/workflows/daily-doc-updater.lock.yml policy_name=job_timeout_minutes_is_required reference=https://github.com/suzuki-shunsuke/ghalint/blob/main/docs/policies/012.md job_name=activation error="job's timeout-minutes is required"
May 22 12:03:22.367 ERR the job violates policies program=ghalint version=1.5.6 workflow_file_path=.github/workflows/daily-doc-updater.lock.yml policy_name=job_timeout_minutes_is_required reference=https://github.com/suzuki-shunsuke/ghalint/blob/main/docs/policies/012.md job_name=agent error="job's timeout-minutes is required"
May 22 12:03:22.367 ERR the job violates policies program=ghalint version=1.5.6 workflow_file_path=.github/workflows/daily-doc-updater.lock.yml policy_name=job_timeout_minutes_is_required reference=https://github.com/suzuki-shunsuke/ghalint/blob/main/docs/policies/012.md job_name=conclusion error="job's timeout-minutes is required"
May 22 12:03:22.367 ERR the job violates policies program=ghalint version=1.5.6 workflow_file_path=.github/workflows/daily-doc-updater.lock.yml policy_name=job_timeout_minutes_is_required reference=https://github.com/suzuki-shunsuke/ghalint/blob/main/docs/policies/012.md job_name=detection error="job's timeout-minutes is required"
vscode ➜ (no) ➜ /workspace (main ✗) $ ghalint run
May 22 12:03:46.009 ERR ghalint failed program=ghalint version=1.5.6 policy_name=job_timeout_minutes_is_required config_file=.github/ghalint.yaml error="validate a configuration file: the policy can't be excluded"
I attempted to bypass this by configuring ghalint.yaml, but certain policies (such as job_timeout_minutes_is_required) are fundamentally designed to not be excludable by default. While this is primarily an issue with how GitHub Agentic Workflows generates its files, I currently have no way to work around it.
```

Would it be possible to allow specifying exclusion rules for specific files or paths? For example:

```yaml
excludes:
  - policy_name: job_timeout_minutes_is_required
    workflow_file_path: .github/workflows/*.lock.yml
    job_name: conclusion
```

```
$ ghalint run
May 22 12:04:06.960 ERR ghalint failed program=ghalint version=1.5.6 policy_name=job_timeout_minutes_is_required config_file=.github/ghalint.yaml error="validate a configuration file: the policy can't be excluded"
```

**Comment by suzuki-shunsuke** [comment]
Some policies support the exclusion, but the policy `job_timeout_minutes_is_required` doesn't support it.

**Comment by y-miyazaki** [comment]
@suzuki-shunsuke 
Thank you for your response.
I understand that it's not supported by design. I'll just ignore the results for now, so I'm closing this issue. Thanks again!
